Graphics/GraphicsEngineWebGPU/include/WebGPUStubs.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 2024 Diligent Graphics LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  In no event and under no legal theory, whether in tort (including negligence),
+ *  contract, or otherwise, unless required by applicable law (such as deliberate
+ *  and grossly negligent acts) or agreed to in writing, shall any Contributor be
+ *  liable for any damages, including any direct, indirect, special, incidental,
+ *  or consequential damages of any character arising as a result of this License or
+ *  out of the use or inability to use the software (including but not limited to damages
+ *  for loss of goodwill, work stoppage, computer failure or malfunction, or any and
+ *  all other commercial damages or losses), even if such Contributor has been advised
+ *  of the possibility of such damages.
+ */
+
+#pragma once
+
+#if PLATFORM_EMSCRIPTEN
+
+#    define WGPUFeatureName_ChromiumExperimentalTimestampQueryInsidePasses 0x000003EE
+
+#    define WGPUFeatureName_Unorm16TextureFormats 0x000003FB
+
+#    define WGPUFeatureName_Snorm16TextureFormats 0x000003FC
+
+#    define WGPUTextureFormat_R16Unorm 0x00000060
+
+#    define WGPUTextureFormat_R16Snorm 0x00000063
+
+#    define WGPUTextureFormat_RG16Unorm 0x00000061
+
+#    define WGPUTextureFormat_RG16Snorm 0x00000064
+
+#    define WGPUTextureFormat_RGBA16Unorm 0x00000062
+
+#    define WGPUTextureFormat_RGBA16Snorm 0x00000065
```

Comment:
How are these constants defined in webgpu headers?
If they are also `defines`, each define need to be wrapped up in `#ifdef` to avoid conflicts.
If they are constants, the constants need to be in `Diligent` namespace to avoid conflicts.